packages/tss-lib/torus.config.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   ...require("@toruslabs/config/torus.config"),
   esm: false,
+  libEsm: false,
+  libCjs: false,
```

Comment:
cjs is false but there is a config for cjs?

Reply:
libCjs and cjs are now 2 different formats exposed by torus-scripts, cjs still stays, i havent removed that.  libCjs  basically creates build using rollup instead of webpack but it doesnt works with wasm currently so i decided to stick with cjs with webpack only.